docs/src/styles/docs/example.scss

```diff
@@ -28,7 +28,11 @@
 .docs-component-demo {
   width: 100%;
   margin-bottom: var(--amplify-space-large);
-
+  
+  & pre[class*=language-] {
+    max-height: none;
```

Comment:
Yup this will only affect the code snippet in the demo block at the top of primitive pages, but there might be a better solution. The issue was they weren't stretching:

<img width="863" alt="CleanShot 2021-11-18 at 09 33 14@2x" src="https://user-images.githubusercontent.com/321279/142466878-a4bfb989-a00d-4372-b5b4-8afcd673b0a4.png">

I could solve that another way though